src/sentry/api/endpoints/group_hashes.py

```diff
@@ -42,3 +45,26 @@ def get(self, request, group):
             order_by='id',
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def delete(self, request, group):
+        id_list = request.GET.getlist('id')
+        if id_list is None:
+            return Response()
+
+        hash_list = GroupHash.objects.filter(
```

Comment:
I'm not totally sure what you mean by status (status of the merge, maybe?) but yeah, there's a lot to be done here to get this to be something thats usable in practice.